modules/luci-mod-rdash/root/usr/share/rpcd/acl.d/luci-mod-rdash.json

```diff
@@ -0,0 +1,53 @@
+{
+	"luci-mod-rdash-routes": {
+		"description": "Grant access to the system route status",
+		"read": {
+			"ubus": {
+				"file": [ "exec" ]
+			}
+		}
+	},
+	"luci-mod-rdash-index": {
+		"description": "Grant access to main status display",
+		"read": {
+			"file": {
+				"/www/luci-static/resources/view/status/include": [ "list" ],
+				"/etc/board.json": [ "read" ],
+				"/proc/sys/net/netfilter/nf_conntrack_count": [ "read" ],
+				"/proc/sys/net/netfilter/nf_conntrack_max": [ "read" ]
+			},
+			"ubus": {
+				"luci": [ "getVersion" ],
+				"file": [ "list", "read" ],
+				"system": [ "board", "info" ],
+				"network.rrdns": [ "lookup" ]
```

Comment:
There does not appear to be any reverse lookup calls in this code, so please drop this permission.